collectors/python.d.plugin/gearman/gearman.chart.py

```diff
@@ -0,0 +1,218 @@
+# Description: dovecot netdata python.d module
+# Author: Kyle Agronick (agronick)
+# SPDX-License-Identifier: GPL-3.0+
+
+# Gearman Netdata Plugin
+
+from bases.FrameworkServices.SocketService import SocketService
+from copy import deepcopy
+
+priority = 60500
```

Comment:
If there is no reason to not use default priority i suggest to remove it.

Reply:
The priority thing is uses for order on the dashboard. If all collectors use default priority then alphabetical order will be used.

Reply:
I was seeing the different Gearman servers in different order depending on the name. Is this preferred or is there a different way to suppress that?

Reply:
> different Gearman

Not sure i understood you correctly, this is the order on the dashboard. 

(almost) all python modules have default priority - default == netdata will use alphabetical.

```
 - system
 ... 
 - apache job(s)
 ...
 - gearmand job(s)
 ...
 - sensors job(s)
```